docs/UserGuide.md

```diff
@@ -382,11 +425,11 @@ Exits the program.
 
 Format: `exit`
 
-### Saving the data
+## Saving the data
```

Comment:
If there is no requirement in the rubrics, I think we can move these under the FAQ.